Python/initconfig.c

```diff
@@ -141,6 +153,17 @@ static const char usage_6[] =
 "   and end column offset) to every instruction in code objects. This is useful \n"
 "   when smaller cothe de objects and pyc files are desired as well as suppressing the \n"
 "   extra visual location indicators when the interpreter displays tracebacks.\n";
+static const char usage_envvars5[] = "\
+These variables have equivalent command-line parameters (see --help for details):\n\
+PYTHONDEBUG             : enable parser debug mode (-d)\n\
+PYTHONDONTWRITEBYTECODE : don't write .pyc files (-B)\n\
+PYTHONINSPECT           : inspect interactively after running script (-i)\n\
+PYTHONNOUSERSITE        : disable user site directory (-s)\n\
+PYTHONOPTIMIZE          : enable level 1 optimizations (-O)\n\
+PYTHONUNBUFFERED        : disable stdout/stderr buffering (-u)\n\
+PYTHONVERBOSE           : trace import statements (-v)\n\
+PYTHONWARNINGS=arg      : warning control (-W arg)\n\
```

Comment:
@warsaw in the end, I wrote a short help summary for each envvar rather than copying the full lines.
Didn’t want to create the burden of writing the same things twice in the same file, with slightly different formatting.

Of course I am ready to change that if reviewers think full details should be here.  There are only 8 envvars, and the text is already duplicated in source, man page and rst docs.